package.json

```diff
@@ -24,72 +24,72 @@
     "mobile"
   ],
   "dependencies": {
-    "bplist-parser": "0.0.6",
-    "broccoli": "0.16.2",
+    "bplist-parser": "0.1.0",
+    "broccoli": "0.16.4",
     "broccoli-funnel": "0.2.3",
     "bufferpack": "0.0.6",
     "bufferutil": "https://github.com/telerik/bufferutil/tarball/master",
-    "byline": "4.1.1",
-    "chalk": "1.0.0",
-    "cli-table": "https://github.com/telerik/cli-table/tarball/master",
-    "colors": "0.6.2",
+    "byline": "4.2.1",
+    "chalk": "1.1.0",
+    "cli-table": "https://github.com/telerik/cli-table/tarball/v0.3.1.1",
+    "colors": "1.1.2",
     "ffi": "https://github.com/icenium/node-ffi/tarball/master",
     "fibers": "https://github.com/icenium/node-fibers/tarball/v1.0.5.1",
-    "filesize": "2.0.3",
+    "filesize": "3.1.2",
     "gaze": "0.5.1",
     "gulp": "3.9.0",
-    "iconv-lite": "0.4.4",
-    "inquirer": "0.8.2",
+    "iconv-lite": "0.4.11",
+    "inquirer": "0.9.0",
     "ios-sim-portable": "1.0.8",
-    "lockfile": "1.0.0",
-    "lodash": "3.6.0",
-    "log4js": "0.6.22",
+    "lockfile": "1.0.1",
+    "lodash": "3.10.0",
+    "log4js": "0.6.26",
     "marked": "0.3.3",
-    "marked-terminal": "1.4.0",
-    "minimatch": "2.0.4",
-    "mkdirp": "0.3.5",
-    "mute-stream": "0.0.4",
+    "marked-terminal": "1.5.0",
+    "minimatch": "2.0.10",
+    "mkdirp": "0.5.1",
+    "mute-stream": "0.0.5",
     "node-inspector": "0.7.4",
-    "node-uuid": "1.4.1",
+    "node-uuid": "1.4.3",
     "npm": "2.6.1",
-    "open": "0.0.4",
-    "osenv": "0.1.0",
+    "open": "0.0.5",
+    "osenv": "0.1.3",
     "plist": "1.1.0",
     "plistlib": "0.2.1",
-    "progress-stream": "0.5.0",
+    "progress-stream": "1.1.1",
     "prompt": "https://github.com/Icenium/prompt/tarball/master",
     "properties-parser": "0.2.3",
     "ref": "https://github.com/icenium/ref/tarball/master",
     "ref-struct": "https://github.com/telerik/ref-struct/tarball/master",
-    "rimraf": "2.2.6",
-    "semver": "4.3.4",
-    "shelljs": "0.3.0",
+    "rimraf": "2.4.2",
+    "semver": "5.0.1",
+    "shelljs": "0.5.1",
     "tabtab": "https://github.com/Icenium/node-tabtab/tarball/master",
-    "temp": "0.8.1",
+    "temp": "0.8.3",
     "through2": "2.0.0",
     "utf-8-validate": "https://github.com/telerik/utf-8-validate/tarball/master",
-    "vinyl-filter-since": "2.0.0",
+    "vinyl-filter-since": "2.0.2",
     "winreg": "0.0.12",
-    "ws": "0.7.1",
+    "ws": "0.7.2",
```

Comment:
https://github.com/websockets/ws/commit/046390f92b0dd59d30664a5be368b5ccb8217314
Due to this change in ws, we have to update our forks of `utf-8-validate` and `bufferutil`